src/pages/Explorer/Aside.svelte

```diff
@@ -1,70 +1,54 @@
 <script>
   import Widget from './Category/Widget.svelte'
   import ExternalLink from './Components/ExternalLink.svelte'
+  import LayoutItem from './Layouts/LayoutItem.svelte'
+  import SocialTrend from './Layouts/SocialTrend.svelte'
+  import WeeklyReport from './Layouts/WeeklyReport.svelte'
+  import SheetsTemplate from './Layouts/SheetsTemplate.svelte'
 
   let className = ''
   export { className as class }
+
+  // TODO: add getItems props for each widget
 </script>
 
 <aside class={className}>
-  <Widget title="Recent Chart Layouts" icon="chart" />
+  <Widget title="Recent Chart Layouts" icon="chart">
+    <svelte:fragment let:item slot="item">
+      <LayoutItem small {item} />
+    </svelte:fragment>
+  </Widget>
```

Comment:
Since the default slot will be used
```suggestion
  <Widget title="Recent Chart Layouts" icon="chart" let:item>
    <LayoutItem small {item} />
  </Widget>
```